cpp/include/cudf/lists/detail/gather.cuh

```diff
@@ -112,6 +107,11 @@ gather_data make_gather_data(cudf::lists_column_view const& source_column,
                       return src_offsets[index];
                     });
 
+  // now that we are doing using the gather_map, we can release the underlying prev_base_offsets.
```

Comment:
Do you mean "done" rather than "doing"?

Reply:
This change is from PR #5816 - sorry for the confusion :( cc @nvdbaranec